README.md

```diff
@@ -1,2 +1,4 @@
 # code-review-test-repo
```

Comment:
Add a comment as part of an actual review __but__ this time from a collaborator, not author.

Reply:
Resolve conversation, to prepare for another test can be done with a review that requests changes.